### PR TITLE
fix(AudioPlayer): render current time and duration

### PR DESCRIPTION
**Pull request by olegshilov**
update readme;

### CODE REVIEW

File: src/components/AudioPlayer/README.md [hunk 1]
```diff
@@ -12,14 +12,21 @@
 ```jsx
 <AudioPlayer
   pending
-  src="http://www.html5tutorial.info/media/vincent.mp3"
+  src="https://www.soundboard.com/handler/DownLoadTrack.ashx?cliptitle=Never+Gonna+Give+You+Up-+Original&filename=mz/Mzg1ODMxNTIzMzg1ODM3_JzthsfvUY24.MP3"
```
Comment (shabanov-dlg) [inline src/components/AudioPlayer/README.md]: image

File: src/components/AudioPlayer/AudioPlayer.css [hunk 2]
```diff
@@ -34,19 +34,32 @@
...
 .playerTime {
-  display: block;
-  letter-spacing: -0.3px;
-  font-size: 13px;
-  line-height: 16px;
+  display: flex;
+  flex-flow: row nowrap;
+  align-content: center;
+  align-items: center;
+  justofy-content: flex-start;
```
Comment (alexboyko) [inline src/components/AudioPlayer/AudioPlayer.css]: typo

File: src/components/AudioPlayer/AudioPlayer.js [hunk 3]
```diff
@@ -11,26 +11,31 @@ import AudioPlayerButton from './AudioPlayerButton/AudioPlayerButton';
 import { PeerInfoTitle } from '../PeerInfoTitle/PeerInfoTitle';
 import styles from './AudioPlayer.css';
 
-type Props = {
+type AudioPlayerProps = {
   src: ?string,
   duration?: ?number,
   pending?: ?boolean,
   sender?: ?string,
+  showDuration: boolean,
```
Comment (alexboyko) [inline src/components/AudioPlayer/AudioPlayer.js]: let's do it optional

File: src/components/AudioPlayer/AudioPlayer.js [hunk 4]
```diff
@@ -11,26 +11,31 @@ import AudioPlayerButton from './AudioPlayerButton/AudioPlayerButton';
 import { PeerInfoTitle } from '../PeerInfoTitle/PeerInfoTitle';
 import styles from './AudioPlayer.css';
 
-type Props = {
+export type AudioPlayerProps = {
   src: ?string,
   duration?: ?number,
   pending?: ?boolean,
   sender?: ?string,
+  showDuration?: boolean,
```
Comment (meafmira) [inline src/components/AudioPlayer/AudioPlayer.js]: Remove optional. U have defaultProps

**Pull request merged**